inventory_management/tests/test_stock_balance_report.py

```diff
@@ -0,0 +1,61 @@
+import frappe
+from frappe.tests.utils import FrappeTestCase
+from inventory_management.inventory_management.report.stock_balance.stock_balance import (
+    get_balance_qty_for_sle,
+)
+from inventory_management.inventory_management.doctype.stock_entry.test_stock_entry import (
+    create_test_parent_child_warehouses,
```

Comment:
since these tests are related to report, they should be inside the report controller folder not in a test folder